### PR TITLE
[BPK-2630] Fix mobile dialog scrim transition bug

### PR DESCRIPTION
**Pull request by georgegillams**

### CODE REVIEW

File: packages/bpk-scrim-utils/src/bpk-scrim.scss [hunk 1]
```diff
@@ -47,9 +47,5 @@
 
   &--appear-active {
     opacity: $bpk-scrim-opacity;
-
-    @include bpk-breakpoint-mobile {
-      opacity: $bpk-scrim-mobile-opacity;
```
Comment (georgegillams) [inline packages/bpk-scrim-utils/src/bpk-scrim.scss]: This value was 1, which is why, on mobile, it was causing it to reach full opacity befoore returning to the final opacity value of 0.7

File: UNRELEASED.yaml [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # Unreleased
 
 > Place your changes below this line.
+FIXED:
```
Comment (ojcurtis) [inline UNRELEASED.yaml]: Can you change this to **Fixed:**

Comment (georgegillams) [reply UNRELEASED.yaml]: I'll leave it as is for now as it's what the release script expects!

**Comment by georgegillams** [comment]
> Can an entry be added to UNRELEASED

Lol - it was but I didn't update it when it became UNRELEASED.yaml 🤦‍♂

**Pull request merged**